WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift

```diff
@@ -550,11 +614,11 @@ struct ReaderPostMenuButtonTitles {
         static let commentFollowError = NSLocalizedString("Could not subscribe to comments", comment: "The app failed to subscribe to the comments for the post")
         static let commentUnfollowError = NSLocalizedString("Could not unsubscribe from comments", comment: "The app failed to unsubscribe from the comments for the post")
         static let unknownSiteText = NSLocalizedString(
-            "reader.notice.follow.site.unknown",
-            value: "this site",
+            "reader.notice.subscribe.site.unknown",
```

Comment:
Should we also update the term in the key to `blog`? 

```suggestion
            "reader.notice.subscribe.blog.unknown",
```

Apart from this entry, some keys in this PR are still using `site`. I'll leave it up to you whether to update them or not since they are not user-facing.

Reply:
I don't have a strong preference one way or the other, so I've updated the keys to `blog` in: https://github.com/wordpress-mobile/WordPress-iOS/pull/22436/commits/2870c4f76986786c8fb13047caefc414430b190a.